src/classiceditorui.js

```diff
@@ -88,6 +99,29 @@ export default class ClassicEditorUI {
 				}
 
 				return Promise.all( promises );
+			} )
+			.then( () => {
+				// Listen on the keystrokes from the main UI.
+				this.keystrokes.listenTo( this.view.element );
+
+				// Listen on the keystrokes from the floating panels, toolbars and the such.
+				this.keystrokes.listenTo( this.view._bodyCollectionContainer );
```

Comment:
I don't think that this handler should listen to keystrokes executed in the body collection. Why? Open http://localhost:8125/ckeditor5-link/tests/manual/link.html, open link balloon, focus the input and press <kbd>Alt+F10</kbd>. The toolbar gets focus, while, IMO, this shouldn't happen. At least not without closing the balloon.

E.g. if you test the same keystroke in CKEditor 4, the toolbar can only be focused if you have a focus in the editable. You can neither focus it if you have focus in a dialog or dropdown. 

So, to me, it seems that <kbd>Alt+F10</kbd> should be handled by the main keystroke handler. The editable is the starting point of all interaction and the toolbar**s** can be focused starting from here only.